app/controllers/ctc/portal/bank_account_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Ctc::Portal::BankAccountController < Ctc::Portal::BaseIntakeRevisionController
-
   def edit
-    @form = form_class.new(current_intake)
+    # Deliberately present the form a blank bank account so as to not leak sensitive details
+    @form = form_class.from_bank_account(BankAccount.new)
```

Comment:
will it save the blank bank account for this client if they don't add any information?

Reply:
Never mind! I see we have validation for not allowing blank fields